website_sale_qty/models/product_template.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from datetime import date
+from openerp import api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    price_quantity_tiers = fields.Serialized(
+        compute='_compute_price_quantity_tiers',
+        help='Unit prices by pricelist and minimum quantity',
+    )
+
+    @api.multi
+    def _compute_price_quantity_tiers(self):
+        pricelists = self.env['product.pricelist'].search([('id', '>', 0)])
+
+        for record in self:
+            results = {}
+
+            for pricelist in pricelists:
+                pricelist_items = self.env['product.pricelist.item'].search([
+                    ('pricelist_id', '=', pricelist.id),
+                    ('product_tmpl_id', '=', record.id),
+                    '|',
```

Comment:
I think that would be logically different. The current set of domains works out to:
```
pricelist_criterion and product_tmpl_criterion 
and (date_start_criterion_1 or date_start_criterion_2) 
and (date_end_criterion_1 or date_end_criterion_2)
```

But with the change, it would be:
```
pricelist_criterion and product_tmpl_criterion 
and ((date_start_criterion_1 or date_start_criterion_2) or date_end_criterion_1)
and date_end_criterion_2
```

My testing seems to confirm this.